### PR TITLE
[openmpi] Update README about GPU and horovod training

### PR DESCRIPTION
**Pull request by jiezhang**
Ref #826

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/831)

### CODE REVIEW

File: kubeflow/openmpi/README.md [hunk 1]
```diff
@@ -79,6 +94,9 @@ Horovod is a distributed training framework for TensorFlow. You may use this pac
 1. Follow the steps in [Quickstart](#quickstart) and use a different entry point.
 
 ```
+# See https://hub.docker.com/r/uber/horovod/tags/ for available tags.
```
Comment (alsrgv) [inline kubeflow/openmpi/README.md]: Let's switch to `uber/horovod:0.13.0-tf1.8.0-torch0.4.0-py2.7`

Comment (jiezhang) [reply kubeflow/openmpi/README.md]: Done.

**Pull request merged**